### DISCUSSION TITLE
Increase storage home assistant VM

### DISCUSSION THREAD

**Discussion by elRadix** [tteck/Proxmox#713] [Help Needed] [answered]
Is it possible to increase the storage disk of a current installed home assistant VM, i'm running out of storage currently?

**Comment by tteck** [comment, marked as answer]
Yep
![Screenshot 2022-11-10 8 02 43 AM](https://user-images.githubusercontent.com/21342670/201099251-4c02c47c-bd1a-41e2-911b-73d34ae0c1dc.png)

**Comment by elRadix** [comment]
and reboot the vm, that's all? nothing else after?

**Reply by tteck** [reply]
That's all! Pretty easy.

**Reply by elRadix** [reply]
thank you for your quick response!
